scripts/odo-functions.sh

```diff
@@ -46,7 +46,7 @@ function odoPush() {
 
 function odoUrl() {
 	echo "- Creating url for odo component: $COMPONENT_NAME" |& tee -a $ODO_BUILD_LOG $ODO_DEBUG_LOG
-	$ODO_CLI url create $COMPONENT_NAME |& tee -a $ODO_BUILD_LOG $ODO_DEBUG_LOG
+	$ODO_CLI url create $COMPONENT_NAME --port 8080 |& tee -a $ODO_BUILD_LOG $ODO_DEBUG_LOG
```

Comment:
Do we always use port 8080?  I think the previous discussion was to only set to 8080 if there are more than one port configured.

Reply:
Actually **always use 8080** and **use 8080 only for multiple ports** are same for us from functionality perspective. I check the config file that generated by odo, all our supported project templates are using `8080` as single port, java project template has multiple ports including `8080`, so `8080` is the default port for odo. I already tested all project templates and no regression for this change.

Reply:
Ya all our currently listed master templates use port 8080, so I think until we put in the functionality where the template defines the app port to use, this shortcut/hardcode is fine

Reply:
Agree with Len, I believe once we enable the project setting on next release then we can remove this temporary workaround and have a better stable design for configuring port and other possible settings.